src/custom-element-registry.ts

```diff
@@ -67,7 +69,7 @@ export class CustomElementRegistry implements ICustomHtmlRegistry {
       classDefinition: classDefinition
     };
 
-    if (tagName.indexOf('-') === -1) {
+    if (this.mandatoryPrefix || tagName.indexOf('-') === -1) {
```

Comment:
what if the element name already starts with the prefixed, do we check that, or do we prefix it again? Example:
- does `bdl-range` become `bdl-bdl-range`?

Reply:
- Good point, currently it is prefixed again. I can consider whether the prefix is already there so compare fallbackprefix with tagname.startsWith. 
- regarding name `mandatoryPrefix`, `alwaysPrefix` sounds good as well, or what about `forcePrefix` - it is boolean, so might be meaningful.

Reply:
I was considering between `always` vs `force`, and `force` will like it's leaning towards more error checking that signaling the effect of this flag. Probably we should consider:
- no ambiguity (force: erring or helping?)
- short (force: shorter but similar length with always)

Reply:
* I put forcePrefix - shorter
* Added check to avoid double prefix au-au-component or custom-custom-element

Reply:
Well, I like your stance, I'm not 100% on `always` so let's go with `force`